package.json

```diff
@@ -37,10 +37,14 @@
     "@babel/preset-react": "^7.18.6",
     "@commitlint/cli": "^17.2.0",
     "@commitlint/config-conventional": "^17.2.0",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
```

Comment:
Is emotion used somewhere? I did a quick search, and looks like these two packages are not used.

Reply:
They need to come with material-ui based on the documentation. I guess some components are using them.

Reply:
they're part of the installation instructions of MUI

Reply:
Ah, I see. Thanks for the clarification.